docs/GettingStarted/configuration/ProductionBestPractice.md

```diff
@@ -1,68 +1,58 @@
 # Customizing Pelorus
 
-See [Configuring the Pelorus Stack](../../configuration/PelorusCore/) for a full readout of all possible configuration items. The following sections describe the  most common supported customizations that can be made to the Pelorus configuration object YAML file.
+See [Configuring the Pelorus Stack](./PelorusCore.md) for a full readout of all possible configuration items. The following sections describe the  most common supported customizations that can be made to the Pelorus configuration object YAML file.
```

Comment:
The second phrase does not make sense for me. I would change to something like

"The following sections describe the most important configuration options of Pelorus, focusing in a production environment."

Reply:
changed.